src/main/java/org/cloudfoundry/promregator/auth/NullEndpointProvider.java

```diff
@@ -0,0 +1,40 @@
+package org.cloudfoundry.promregator.auth;
+
+import static com.sap.cloud.security.xsuaa.Assertions.assertNotNull;
+
+import java.net.URI;
+
+import com.sap.cloud.security.config.CredentialType;
+import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.xsuaa.client.OAuth2ServiceEndpointsProvider;
+
+public class NullEndpointProvider implements OAuth2ServiceEndpointsProvider {
```

Comment:
I don't like the name of the class: it's not "a null-endpoint" that is returned, but it's just that we provider an adapter to our type world.
What about `PromregatorOAuth2ServiceEndpointsProvider`?

Reply:
Done. The naming was inspired by the NullEnricher.